### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by wadaudzai**
Hello Dear Reviewer.

Below are the tasks that were completed while working on this project.
- React App initialized.
- Directory was created for components and new components were defined.
- React router added and set for app's navigation.
- Components created like displaying the list of books, displaying a book and a form with inputs but no functionality yet.
- Created a button in the categories section.

Dear Reviewer I have worked on this project to the best of my knowledge please check it if you got time. Thanks for your time.

Regards, 

Daudzai

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+
+### Deployment
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
Comment (ahmetbozaci) [inline README.md]: - [ ] Kindly update your README file. 
You can use this [template](https://github.com/microverseinc/readme-template) 
and make sure there are no broken links(screenshot, license, etc.) and empty sections. 
Kindly change the readme template's sentence with your sentence.

File: src/Styling/Book.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  align-items: center;
+}
+
+.book-info-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.book-actions-container {
+  display: flex;
```
Comment (ahmetbozaci) [inline src/Styling/Book.css:9]: - [ ] For DRY CSS kindly combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Please check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/Book.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import '../Styling/Book.css';
+
+const Book = (props) => {
+  /* eslint-disable react/prop-types */
```
Comment (ahmetbozaci) [inline src/components/Book.js]: - [ ] Kindly don't disable the linter, if it is giving some error please fix it.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️

 @wadaudzai  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
✔️ Your git-flow is correct. 
✔️ You set up linter files and no errors. 
✔️ Your pull request is very professional. It has a meaningful title and summary. Keep it up.
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @wadaudzai 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**